users.html

```diff
@@ -509,6 +542,9 @@ <h1 id="page-title" ng-non-bindable="">GEOS-Chem Users</h1>
   // National Technical University of Ukraine (KPI)
   L.marker([50.448890539716245, 30.45694481095485], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>National Technical University of Ukraine (KPI)</strong><br>Kyiv, Ukraine<ul><li><a href='https://kpi.ua/' target='_new'>Website</a></li></ul>");
 
+  // National University of Defense Technology
+  L.marker([38.86676881818191, -77.01445037946115], { icon: blueIcon }).addTo(mymap).bindPopup("<strong>National University of Defense Technology</strong><br>Changsha, China<ul><li><a href='http://www.ndu.edu/' target='_new'>Website</a></li></ul>");
```

Comment:
These coordinates are correct: 28.213807421292195, 112.99362999658231

Reply:
Also ndu.edu is the National Defense Unversity in Washington DC.   The proper link should be: https://english.nudt.edu.cn/

Reply:
Thanks for catching that. The wrong location showed up when I searched in Google Maps. Now fixed in https://github.com/geoschem/geoschem.github.io/pull/44/commits/b4294767881fd7c818b0732b91daebb180bf9f02.